src/donut/dbxray/generate/datapotato.cljc

```diff
@@ -0,0 +1,43 @@
+(ns donut.dbxray.generate.datapotato
+  (:require
+   [clojure.string :as string]
+   [donut.dbxray.generate :as ddg]))
+
+(defn- gen-table-prefix
+  [table-name]
+  (let [words (string/split (name table-name) #"_")
+        initials (map (comp str first) words)]
+    {:prefix (keyword (apply str initials))}))
+
+(defn- add-table-prefix-to-col-name
+  [table-name col-name]
+  (let [table (name table-name)]
+    (keyword (str table "/" (name col-name)))))
```

Comment:
elsewhere I have this as

```clojure
(defn- column-spec-name
  [table-name column-name]
  (keyword (name table-name) (name column-name)))
```

the name `column-spec-name` doesn't quite make sense here though, so maybe `full-column-name`?